src/lib/components/drs/formComponents/VerifyIdButton.js

```diff
@@ -43,6 +43,12 @@ const VerifyIdButton = props => {
         }
     }
 
+    useEffect (() => {
```

Comment:
why is this needed? Should the verify Id module behavior change based on whether the form is `new` or `edit`? The `useEffect` will cause `callAndValidate` to run when the component is rendered on the screen for the first time. If it's a button, `callAndValidate` should be run upon a button click